scripts/build_validated_credential_supply_map.py

```
@@ -89,7 +91,171 @@ def _to_repo_relative_posix(path: Path) -> str:
+    )
+
+
+def _redact_cli_path_text(
+    value: object,
+    raw_paths: Sequence[str],
+    native_paths: Sequence[Path],
+) -> str:
+    """Redact raw and host-native CLI path forms from caught diagnostics."""
+    text = str(value)
+    replacements: Dict[str, str] = {}
+    for raw_path, native_path in zip(raw_paths, native_paths):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)